spring-cloud-sleuth-core/pom.xml

```diff
@@ -32,6 +32,11 @@
 	</parent>
 
 	<dependencies>
+		<dependency>
```

Comment:
I don't think this can be a mandatory dependency.

Reply:
yeah, after thinking about this, I can make this a test scope dependency, as it is during the tests that the failures manifest. Still not a great solution, but I am not sure what else would make more sense?